tests/test_generic/tests/test_xml.py

```diff
@@ -470,3 +471,41 @@ def _check_dates_are_relative(self, root, file_name):
                         model_name,
                         file_name,
                     )
+
+    def _check_static_values_in_inputs(self, root, file_name):
+        allowed_inputs = {"resourceCapacity"}
+        for tag in root.xpath("//input[@value] | //option[@value]"):
+            value = tag.get("value")
+            if not (value and value.isdigit()):
+                continue
+            if tag.get("t-att-value"):
+                continue
+            if tag.get("name") in allowed_inputs:
+                continue
+
+            line = getattr(tag, "sourceline", "?")
+            _logger.warning(
+                "Static value '%s' found in <%s name='%s'> in %s (line %s). "
+                "Please use t-att-value=\"request.env.ref('module.record').id\" instead.",
+                value,
+                tag.tag,
+                tag.get("name") or tag.get("id") or "",
```

Comment:
```suggestion
            name = tag.get("name")
            if name in allowed_inputs:
                continue

            line = getattr(tag, "sourceline", "?")
            _logger.warning(
                "Static value '%s' found in <%s name='%s'> in %s (line %s). "
                "Please use t-att-value=\"request.env.ref('module.record').id\" instead.",
                value,
                tag.tag,
                name or tag.get("id") or "",
```
No need to retrieve name more than once.